controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller.go

```diff
@@ -216,11 +220,32 @@ func (r *Reconciler) Reconcile(ctx goctx.Context, req ctrl.Request) (_ ctrl.Resu
 		VMPublishRequest: vmPublishReq,
 	}
 
+	skipPatch := false
```

Comment:
Move this to L228 (close to where this is referenced).

Reply:
I would also redefine this as:

```golang
var skipPatch bool
```

It's just preference, but since the default value is `false` anyway, it looks cleaner (to me). While at it, I would love to see some documentation:

```golang
var skipPatch bool // the patch may be skipped when the following
                   // conditions occur:
                   //
                   //     * it is rainy outside
                   //     * the patch just is not feeling it today
                   //     * the application failed due to gremlins in the system
                   //     * etc.
```